### PR TITLE
SXT Website: More examples

### PR DESCRIPTION
**Pull request by michalsanger**

### CODE REVIEW

File: src/sx-tailwind-website/src/TailwindCss/Buttons.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+// eslint-disable-next-line no-unused-vars
+import * as sx from '@adeira/sx';
```
Comment (tbergq) [inline src/sx-tailwind-website/src/TailwindCss/Buttons.js:5]: Why do you need to import this?

**Review by mrtnzlml — APPROVED** [review]
I had and still have the same question as @tbergq but otherwise 👍

File: src/sx-tailwind-website/src/TailwindCss/Buttons.js (hunk 1 above)
Comment (michalsanger) [reply src/sx-tailwind-website/src/TailwindCss/Buttons.js:5]: It's a workaround until I find out what is going on during the Next.js build. It was failing with `ReferenceError: sx is not defined`. There are some optimisation happening, source code of imported components is pasted into code that was importing them and for some reason it does not play well with that Tailwind to SX Babel transformation.

Do you have any idea what could be the issue? cc @mrtnzlml

Comment (michalsanger) [reply src/sx-tailwind-website/src/TailwindCss/Buttons.js:5]: Fixed in https://github.com/adeira/universe/pull/1220

**Pull request merged**